TestAssets/TestProjects/StandaloneApp/project.json

```diff
@@ -0,0 +1,42 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc3-*"
+      }
+    }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "win8-x64": {},
+    "win8-x86": {},
+    "win81-x64": {},
+    "win81-x86": {},
+    "win10-x64": {},
+    "win10-x86": {},
+    "osx.10.10-x64": {},
+    "osx.10.11-x64": {},
+    "ubuntu.14.04-x64": {},
+    "ubuntu.16.04-x64": {},
+    "centos.7-x64": {},
+    "rhel.7.2-x64": {},
+    "debian.8-x64": {},
+    "fedora.23-x64": {},
+    "opensuse.13.2-x64": {}
+  },
+  "runtimeOptions": {
```

Comment:
Are these used in the tests?  If not, I'd prefer not to have "gunk" in the project.json.